prac_05/word_occurrences.py

```diff
@@ -6,6 +6,19 @@
 user_text = input(str("Please enter a string of text: "))
 
 words = user_text.split()
+#Counting the words from the users input
+for word in words:
+    word_occurrence = users_words.get(word, 0)
```

Comment:
Maybe think about re-naming some of your variables. Although I understand why you've named them what you have because you've used word in so many of them it can be difficult to follow which one you're using.

Reply:
Yeah, it did get a bit confusing. Thank you for your input.

Reply:
Updated code now with better names - avoiding word and words in anything other than those two. 
Thank you.